tigerpath/majors_and_certificates/majors/AAS_2018.json

```
@@ -0,0 +1,815 @@
+{
+  "type": "Major",
+  "name": "African American Studies",
+  "code": "AAS",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-african-american-studies",
+    "https://aas.princeton.edu/academics/undergraduate-program/concentration-requirements"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Eddie S. Glaude Jr.",
+      "email": "esglaude@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Naomi Murakawa",
+      "email": "murakawa@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Imani Perry",
+      "email": "iperry@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Prerequisite for entry into the African American Studies concentration is the successful completion on a graded basis of at least one core survey African American Studies (AAS) course.",
+      "double_counting_allowed": true,
+      "course_list": [
+        "AAS 245",
+        "AAS 353",
+        "AAS 359",
+        "AAS 366",
+        "AAS 367"
+      ]
+    },
+    {
+      "name": "Core Survey",
+      "max_counted": 1,
+      "min_needed": 2,
+      "explanation": "Students complete two (2) core survey courses listed below. At least one (1) of these must be a Pre-20th century course. Students are strongly encouraged to complete both survey courses by the end of junior year.",
+      "req_list": [
+        {
+          "name": "Pre-20th Century",
+          "max_counted": null,
+          "min_needed": 1,
+          "course_list": [
+            "AAS 353",
+            "AAS 366",
+          ]
+        },
+        {
+          "name": "20th Century and Beyond",
+          "max_counted": 1,
+          "min_needed": null,
+          "course_list": [
+            "AAS 245",
+            "AAS 359",
+            "AAS 367"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Track",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Concentrators are required to complete nine (9) courses: two (2) core survey courses, AAS 300 Junior Seminar: Research and Writing in African American Studies, and six (6) additional African American Studies courses. In addition, all concentrators are required to participate in the senior colloquium.",

```

I think the explanation here should be about the track structure, right?

Perhaps
```suggestion
      "explanation": "<p>At the end of their fall semester, juniors declare a subfield to pursue, selecting from the following courses:</p><ul><li>African American Culture and Life (AACL)</li><li>Global Race and Ethnicity (GRE)</li><li>Race and Public Policy (RPP)</li></ul><p>Four (4) courses must be taken in the chosen subfield, with two (2) additional courses as follows:</p><ul><li>If the chosen subfield is AACL or RPP, then the two (2) additional courses must be selected from the GRE courses.</li><li>If the chosen subfield is GRE, then one (1) must be an AACL course, and one (1) must be an RPP course.</li></ul><p>Students are permitted to take <em>up to two (2)</em> approved cognate courses in other departments. See the <a href=\"https://aas.princeton.edu/courses\" target=\"_blank\" rel=\"noopener\">departmental website</a> for the lists of courses and approved cognates by subfield.</p>",

```